src/modules/ens/utils.ts

```diff
@@ -64,6 +64,6 @@ export function getNameFromDomain(domain: string): string {
   return domain.split('.')[0]
 }
 
-export function isAllowed(allowance: string) {
-  return Number(allowance) > 100
+export function isEnoughClaimMana(mana: string) {
+  return Number(mana) > 100
```

Comment:
If the authorization and the balance of MANA is `>=`, is enough to claim a name.

Reply:
Also, just to not lose it (it is in an old comment). Should we need to change `100` to `100 * `1e18`?

Reply:
I checked and both the allowance and `wallet.mana` already had the number divided by `1e18`